lib/content-services/src/lib/content-node-share/services/shared-links-api.service.ts

```diff
@@ -65,8 +65,8 @@ export class SharedLinksApiService {
      * @param options Options supported by JS-API
      * @returns The shared link just created
      */
-    createSharedLinks(nodeId: string, options: any = {}): Observable<SharedLinkEntry> {
-        const promise = this.sharedLinksApi.createSharedLink({ nodeId }, options);
+    createSharedLinks(nodeId: string | SharedLinkBodyCreate, options: any = {}): Observable<SharedLinkEntry> {
```

Comment:
`nodeId` should always be a string, if you need additional data you can add new param here